composer.json

```diff
@@ -93,27 +91,32 @@
         "prestashop/statssearch": "^2",
         "prestashop/statsstock": "^2",
         "prestashop/statsvisits": "^2",
-        "prestashop/translationtools-bundle": "^3",
+        "prestashop/translationtools-bundle": "dev-sf4-support",
         "prestashop/welcome": "^6",
-        "sensio/distribution-bundle": "^5.0.19",
-        "sensio/framework-extra-bundle": "^5.0.0",
+        "sensio/distribution-bundle": "^5.0",
+        "sensio/framework-extra-bundle": "^5.1",
         "shudrum/array-finder": "^1.1.0",
         "smarty/smarty": "^3.1.31",
         "soundasleep/html2text": "^0.5.0",
-        "swiftmailer/swiftmailer": "~5.4.3",
+        "swiftmailer/swiftmailer": "~6.2",
```

Comment:
Do we need to update this package?

Reply:
yes, but not because of PHP requirements but because this major release fixes bug we had in the core and that we couldn't fix. I can't remember the related issues, but @PierreRambaud probably do? Also, as you can see, this upgrade has no impact at all on our public API :)

Reply:
It does not supprot recent version of the email adresse RFC, have many bugs with punycode and validators.
But v5.4.3 requires php: >=5.3.3 where 6.2.1 requires php: >=7.0.0, I think coule have some issues with 7.3 if we don't update

Reply:
👍